src/target/imxrt.c

```diff
@@ -379,6 +501,7 @@ static void imxrt_spi_exec_sequence(
 
 static void imxrt_spi_wait_complete(target_s *const target)
 {
+	const imxrt_priv_s *const priv = (imxrt_priv_s *)target->target_storage;
```

Comment:
This is unused in this function, please remove it or use it.

Reply:
This is a consequence of IMXRT_FLEXSPI macro using it. It is used, but not obviously. I'm happy to entertain better ways to do this.